forge-gui/res/editions/Conspiracy Take the Crown.txt

```diff
@@ -5,7 +5,7 @@ Name=Conspiracy: Take the Crown
 Code2=CN2
 Type=Draft
 BoosterCovers=3
-Booster=10 Common:!fromSheet("CN2 Draft Matters"), 3 Uncommon:!fromSheet("CN2 Draft Matters"), 1 RareMythic:!fromSheet("CN2 Draft Matters"), 1 fromSheet("CN2 Draft Matters")
+Booster=10 Common:!fromSheet("CN2 Draft Matters"), 3 Uncommon:!fromSheet("CN2 Draft Matters"), 1 RareMythic:!fromSheet("CN2 Draft Matters"):fromSheet("CN2 cards"), 1 fromSheet("CN2 Draft Matters")
```

Comment:
I Don't understand this change?

Reply:
It is only to not catch the promo card 
![{7C2770E5-F756-4CC2-92F6-C65489EC32CB}](https://github.com/user-attachments/assets/360b6590-7977-49c1-a1f4-b6433e6aa1f8)
Otherwise Kaya as 100% more chance to shows up in a pack I guess.